src/test/java/seedu/address/logic/commands/DeleteCommandTest.java

```diff
@@ -29,7 +28,7 @@
     @Test
     public void execute_validIndexUnfilteredList_success() {
         Person personToDelete = model.getFilteredPersonList().get(INDEX_FIRST_PERSON.getZeroBased());
-        DeleteCommand deleteCommand = new DeleteCommand(INDEX_FIRST_PERSON);
+        DeleteCommand deleteCommand = new DeleteCommand(personToDelete.getName().toString());
```

Comment:
I think? this is fine, but I'm not certain. Double check best practice maybe?

Reply:
Maybe put in a variable with a descriptive name first like `String nameOfPersonToDelete`